src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallerFactory.cs

```diff
@@ -34,9 +35,9 @@ internal class WorkloadInstallerFactory
                 {
                     throw new InvalidOperationException(LocalizableStrings.OSDoesNotSupportMsi);
                 }
-
+                // TODO: should restoreActionConfig be flowed through to the client here as well like it is for the FileBasedInstaller below?
```

Comment:
this is unrelated to this PR, but @dsplaisted I saw a mismatch between the use of RestoreActionConfig in this codepath vs the codepath directly below - should those be synchronized? `NetSdkMsiInstallerClient.Create` has an optional `restoreActionConfig` parameter that we're not flowing at the moment, and I know RAC has been a hot topic recently.

Reply:
Yes, there are cases for tool installation where we're not flowing command line options to a RestoreActionConfig.  If we support those command line options for workload commands it seems like we should flow them through via a RestoreActionConfig.